experimenter/experimenter/nimbus-ui/src/styles/index.scss

```diff
@@ -135,6 +135,7 @@ select.form-control.is-warning {
   height: 18px;
 }
 
+
```

Comment:
oops extra line here

Reply:
Ohh, I thought `make code_format` might have taken care of this, I'll fix it no issues